pkg/deploy/docker_test.go

```diff
@@ -32,6 +33,6 @@ func Test_parseCmd(t *testing.T) {
 }
 
 func Test_runShellCmd(t *testing.T) {
-	err := runShellCmd("echo", "hello")
+	err := runShellCmd(os.Stdout, os.Stderr, "echo", "hello")
```

Comment:
Now that you've added the out/err buffers we could have better integration tests with the shell. 
For example here you could pass an out buffer and check that the string hello is contained in it.